s4-api-mock.http

```diff
@@ -0,0 +1,4 @@
+PATCH http://localhost:8080/api/API_BUSINESS_PARTNER/A_BusinessPartnerAddress(BusinessPartner='10401010',AddressID='200')
```

Comment:
What is this file for?

Reply:
This file provides an example request that can be executed using the VS Code Rest Client plugin (https://marketplace.visualstudio.com/items?itemName=humao.rest-client). It simulates a change of the address on the mocked Business Partner API and triggers the mocked `BO.BusinessPartner.Changed` event, defined in `ApiBusinessPartnerEventMockHandler`. This in turn allows to mock the event receiving in a local scenario to test the corresponding event handler defined in `AdminServiceAddressHandler`.